source/includes/steps-open-new-playground.yaml

```diff
@@ -0,0 +1,34 @@
+title: Open the Visual Studio Code :guilabel:`Command Palette`.
+level: 4
+ref: open-cmd-palette
+content: |
+    In Visual Studio Code, press either
```

Comment:
press one of the following:

Reply:
I'm not sold on this change. To me "Press one of the following" reads like we are presenting them with options and they have to choose one of them. This is just to clarify the different commands for Windows vs macOS.

Do you feel that the current wording is unclear?

Reply:
Actually, I can update to say "press one of the following key combinations"